src/pages/SponsortoEventRedirect.jsx

```diff
@@ -0,0 +1,16 @@
+// Importing Components from node_modules
+import React, {useEffect} from 'react';
+
+const SponsortoEventRedirect = () => {
+    useEffect(() => {
+        window.location.href = 'https://zooengg.ca/calendar';
+      }, []);
+    return (
+
+    <div className="page--body sponsors">
+      <h1>This page will redirect to our events page</h1>
```

Comment:
Can this just be "Redirecting..." with a possible load icon since sometimes this page actually gets seen by users

Reply:
ok!